LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 CodeForPoznan
```

Comment:
Code for Poznan?

As in: `Fundacja pod nazwą Code for Poznań, zwana dalej „Fundacją”,(...)`

Reply:
@w1stler is that what you've meant?